### PR TITLE
Revamp content

### PR DESCRIPTION
**Pull request by bsmithgall**
### What Changed
- Moved footer to the bottom
- Cleaned up the index page
- Added some more ability for customization with pre-button-text
- Added some additional content and context
- Added "Don't know" button that links to [wexplorer](bit.ly/wexplorer) ([github](github.com/codeforamerica/wexplorer))
### Issues
#2 : Go through all the site's content to make sure it's accurate
### Screencap

new-content

### CODE REVIEW

File: _posts/2015-01-01-explanatory-process.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ layout: question
 title: "Use the Explanatory Process"
 origin:
   -
-    text: "Is the item's cost under $2000?"
+    text: "Is the item's cost under $2,000?"
     url: goods-services
 ---
+
+The Explanatory Process allows you to purchase an item whose total cost is $2,000 or less without a contract. However, this purchase must receive *pre-approval* from both the Director of your department and the Director of the Office of Management and Budget, and final approval from City Council.
```
Comment (hamhands) [inline _posts/2015-01-01-explanatory-process.md:10]: This is good copy

File: _posts/2015-01-01-is-it-sole-sourced.md [hunk 2]
```diff
@@ -13,3 +13,7 @@ origin:
     text: "Is it an emergency (professional services)?"
     url: professional-services
 ---
+
+According to a 2009 City of Pittsburgh policy document, 
+
+> a **Sole Source Professional Services contract** is a contract involving unique professional services that are documented to be available from one source only. An example could be art restoration by a particular artist on his or her own work. The term may also include specialized training or maintenance services on verified sole-source purchases.
```
Comment (hamhands) [inline _posts/2015-01-01-is-it-sole-sourced.md:19]: More good copy

File: _posts/2015-01-01-phone-quote-service.md [hunk 3]
```diff
@@ -6,3 +6,7 @@ origin:
     text: "Will it be multiple purchases or uses over time?"
     url: multiple-purchases
 ---
+
+In order to do a phone quote, you must provide the Office of Management and Budget's procurement team with detailed specifications and contact information for at least three likely bidders.
```
Comment (hamhands) [inline _posts/2015-01-01-phone-quote-service.md:10]: Helpful, dog. Helpful.

File: _posts/2015-01-01-phone-quote-single-use.md [hunk 4]
```diff
@@ -3,6 +3,10 @@ layout: question
 title: "Use the phone quote process."
 origin:
   -
-    text: "Is the item's cost over $3000?"
+    text: "Is the item's cost over $30,000?"
     url: over-30-thousand
 ---
+
+In order to do a phone quote, you must provide the Office of Management and Budget's procurement team with detailed specifications and contact information for at least three likely bidders.
```
Comment (hamhands) [inline _posts/2015-01-01-phone-quote-single-use.md:10]: YES. YES.

**Comment by hamhands** [comment]
LGTM :+1:

File: _posts/2015-01-01-explanatory-process.md (hunk 1 above)
Comment (bsmithgall) [inline _posts/2015-01-01-explanatory-process.md:10]: image

File: index.html [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
-    text: "Yes"
-    url: use-it
+    text: "Get Started!"
+    url: is-it-on-contract
+pre_button_content:
   -
-    text: "No"
-    url: what-are-you-sourcing
+    title: "What is this?"
+    content: "This tool is intended to help you navigate which process you should use to purchase an item for the City of Pittsburgh. Just answer each question in sequence until you end up at a recommendation page. Occasionally, it will also link you to outside resources to help you make a decision, answer a question, or learn more about a specific process."
```
Comment (msknee) [inline index.html:13]: "Wondering what steps to take to purchase something for the City of PIttsburgh? Answer a couple quick questions and find out what purchasing process to use."

Comment (bsmithgall) [inline index.html:13]: That's better, will revert and update.

**Pull request merged**